infrastructure/src/main/java/org/corfudb/infrastructure/LayoutServer.java

```diff
@@ -141,11 +148,6 @@ private boolean isBootstrapped(RequestMsg requestMsg) {
     @RequestHandler(type = RequestPayloadMsg.PayloadCase.LAYOUT_REQUEST)
     public void handleLayoutRequest(@Nonnull RequestMsg req, @Nonnull ChannelHandlerContext ctx,
```

Comment:
Since the class is not thread safe its better to disallow access to the handlers. You can mark them as private and annotate with `VisibibleForTesting`